packages/components/src/components/dropdown/dropdown.stories.ts

```diff
@@ -0,0 +1,303 @@
+import { userEvent } from '@storybook/testing-library';
+import { waitUntil } from '@open-wc/testing-helpers';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+const { argTypes, parameters } = storybookDefaults('sd-dropdown');
+const { generateTemplate } = storybookTemplate('sd-dropdown');
+const { overrideArgs } = storybookHelpers('sd-dropdown');
+
+export default {
+  title: 'Components/sd-dropdown',
+  component: 'sd-dropdown',
+  args: overrideArgs([
+    { type: 'slot', name: 'trigger', value: '<sd-button slot="trigger">Trigger</sd-button>' },
+    {
+      type: 'slot',
+      name: 'default',
+      value: '<div class="slot slot--border slot--background"></div>'
+    },
+    {
+      type: 'attribute',
+      name: 'open',
+      value: true
+    }
+  ]),
+  argTypes,
+  parameters: { ...parameters },
+  decorators: [
+    withActions,
+    (story: any) =>
+      html`<style>
+          div.slot {
+            width: 24px;
+            height: 24px;
+          }
+          sd-dropdown:not([rounded]) sd-button::part(base),
+          sd-dropdown:not([rounded]) .slot {
+            border-radius: 0;
+          }
+          td.template,
+          .example,
+          div.example {
+            width: 120px;
+            height: 70px;
+          }
+          .example {
+            display: flex;
+            flex-direction: column;
+            padding: 12px;
+          }
+          .template-placement sd-dropdown[placement^='top'] .slot,
+          .template-placement sd-dropdown[placement^='bottom'] .slot {
+            width: 48px;
+          }
+
+          .template-placement sd-dropdown[placement^='left'] .slot,
+          .template-placement sd-dropdown[placement^='right'] .slot {
+            height: 36px;
+          }</style
+        >${story()}`
+  ] as unknown
+};
+
+/**
+ * Default: This shows sd-dropdown in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return html`<div style="height: 200px; overflow: hidden;">
+      ${generateTemplate({
+        constants: {
+          type: 'slot',
+          name: 'default',
+          value: '<div class="example"></div>'
+        },
+        args
+      })}
+    </div>`;
+  }
+};
+
+/**
+ * For round triggers, set the `rounded` attribute. this automatically adds a minimal distance of 1px between the trigger and the panel.
+ */
+
+export const Rounded = {
+  parameters: { controls: { exclude: ['rounded', 'default'] } },
+  render: (args: any) =>
+    generateTemplate({
+      axis: {
+        x: {
+          type: 'attribute',
+          name: 'rounded'
+        }
+      },
+      constants: {
+        type: 'slot',
+        name: 'default',
+        value: '<div class="example"></div>'
+      },
+      args
+    })
+};
+
+/**
+ * The overlay automatically adjusts its size to fit the screen. The content gets scrollable then. To disable this behavior, set the `no-auto-size` attribute.
+ */
+
+export const NoAutoSize = {
```

Comment:
in the docs page it looks very broken:
![Bildschirmfoto 2023-08-29 um 17 13 14](https://github.com/solid-design-system/solid/assets/48211526/28ebee6b-26a1-4ab3-8ebb-8a83bd707b35)

Reply:
Even after reload?  In fact it's absolutely correct. That's we set `auto-size` to true per default. It resizes itself on the right hand side, but breaks the behaviour on the left-hand side, making the whole page bigger.

It searches for it's best position – and that's at the top, as there is more space available than at the bottom. Indeed I could resize the whole div, so there is more space at the bottom.

Reply:
I improved the story to have minimum-height in the docs! :) Should look better now as soon as it's fully visible.

Reply:
yes, as soon as it is fully visible. 
While scrolling it still feels very buggy and does not give a good impression of the component... 

can we remove specific stories from the docs and just add a NOTE that this story was excluded due to presentation issues and users should go into the story itself?

Reply:
Mh – it just shows the correct behaviour, that it looks for a place to show up and people should be aware of that...

...but I would do something differently: I would add a `no-flip` property, so people can opt-out of the automatic flipping behaviour. And we just set this to true in the problematic stories! :)

Reply:
Added the attribute. Feels absolutely better now. I didn't add no-flip thing to every story, as we still should the default behaviour.